builder/common/artifact.go

```diff
@@ -119,3 +126,18 @@ func (a *Artifact) stateAtlasMetadata() interface{} {
 
 	return metadata
 }
+
+func (a *Artifact) statePARMetadata() interface{} {
```

Comment:
Updating the name like Megan did for the Packer code
```suggestion
// stateHCPPackerRegistryMetadata will write the metadata as an hcpRegistryImage for each of the AMIs 
// present in this artifact. 
func (a *Artifact) stateHCPPackerRegistryMetadata() interface{} {
```